.github/workflows/ci-continuous-integration.yml

```diff
@@ -1,6 +1,10 @@
 on:
   pull_request:
     types: [opened, synchronize]
+    branches:
```

Comment:
if i understand this correctly we only run e2es when PRs are going into develop/release/epic?
is this as intended and is there any way to run E2Es when trying to merge a PR into a non develop branch? 
for example this PR: https://github.com/SAP/spartacus/pull/16962